docs/_templates/footer.html

```diff
@@ -0,0 +1,67 @@
+{%- block footer -%}
+<footer>
+    <div class="frame frame-ruler-before frame-background-dark">
+        <div class="frame-container">
+            <div class="frame-inner">
+
+                {%- if theme_docstypo3org %}
+                {%-
+                    if theme_project_home
+                    or theme_project_contact
+                    or theme_project_discussions
+                    or theme_project_issues
+                    or theme_project_repository
+                %}
+                <ul class="footer-simplemenu">
+                    {%- if theme_project_home %}
+                    <li><a href="{{ theme_project_home|e }}" title="Home"><span>Home</span></a></li>
```

Comment:
Could we remove all those `title` attributes? Realise this isn’t used currently but really these shouldn’t be there.